cores/esp32/esp32-hal-uart.c

```diff
@@ -107,17 +138,45 @@ void uartEnableInterrupt(uart_t* uart)
     uart->dev->int_ena.rxfifo_tout = 1;
     uart->dev->int_clr.val = 0xffffffff;
 
+    if(arg != NULL){
+        (*arg) = malloc(sizeof(uart_interrupt_t));
+        (*arg)->func = func;
+        (*arg)->dev = uart;
+        (*arg)->user_arg = user_arg;
+		
+		if(_uart_interrupt_array[uart->num])
```

Comment:
please keep the style and add brackets. Also something went wrong with tabs/spaces. We use 4 spaces for tab